doc/important_details.rst

```diff
@@ -27,6 +27,13 @@ from segments is natural so this notion of segment is the same in tridesclous.
 This means that you can feed tridesclous with a list a files (with same channels maps of course)
 and tridesclous will generate them as a list of segment naturally.
 
+Note that the raw file should store the data in this order:
+*t0c0 t0c1 t0c2 t0c3... t1c0 t1c1 t1c2 t1c3* where *t{i}c{j}* 
+is sample no. *i* on channel no. *j*. Thus if the data is loaded in a
+2D numpy array `x` where each row is the time series data from one
+channel, you can save it in a raw file with this code:
+`x.T.tofile(filename)`.
+
 
```

Comment:
I am OK with this.
This is not related to Multi segment stuff.

I would add "Information on formats" section. that could be enhanced with neo consideration.